sources/mep-kilogram-definition-en.adoc

```diff
@@ -152,37 +152,37 @@ Since the experimental value of the physical constant stem:[h//m](^28^Si) is kno
+The BIPM in coordination with the CCM organizes an on-going BIPM key comparison <<cipm-mra>>, CCM.M-K8 <<m-k1>>, for laboratories with primary realization methods. In this comparison, the primary mass standards of the participants are compared to artefacts from the BIPM ensemble of reference mass standards (see <<sec-4-2,nosee%>>). The CCM decides the required periodicity of laboratory participation in CCM.M-K8 in order to support relevant calibration and measurement capabilities (CMCs).
 
 In cases where compliance with the CIPM Mutual Recognition Arrangement (CIPM MRA) is required <<cipm-2009-24>>, it is essential that the mass standards are traceable to primary mass standards of a participant in BIPM.M-K1 that has relevant CMC entries or, in the case of the BIPM, suitable entries in its calibration and measurement services as approved by the CIPM. Dissemination of the whole mass scale is validated for all NMIs/DIs and the BIPM through the traditional types of key comparisons organized prior to the present definition of the kilogram. +
 Results of all key comparisons are published in the Key Comparison Database (KCDB) in accordance with the rules of CIPM MRA <<cipm-mra>> and may be used in support of NMI/DI claims of its calibration and measurement capabilities and the BIPM claims listed in its calibration and measurement services.
 
 [[sec-4-2]]
 === Dissemination from the BIPM ensemble of reference mass standards
 
-In accordance with Resolution 1 of the 24^th^ meeting of the General Conference on Weights and Measures (CGPM) (2011) [4.8] and Resolution 1 of the 25^th^ meeting of the CGPM (2014) <<cgpm-1>>, the BIPM maintains an ensemble of reference mass standards "`_to facilitate the dissemination of the unit of mass_`" in the revised SI. This ensemble is presently composed of eighteen 1 kg artefacts of various materials which have been chosen to minimize known or suspected sources of mass instability. A storage facility has been designed to minimize the rate of surface contamination of the artefacts. Technical details are provided in <<bipm-paper>>.
+In accordance with Resolution 1 of the 24th meeting of the General Conference on Weights and Measures (CGPM) (2011) <<cgpm-24>> and Resolution 1 of the 25th meeting of the CGPM (2014) <<cgpm-1>>, the BIPM maintains an ensemble of reference mass standards "`_to facilitate the dissemination of the unit of mass_`" in the revised SI. This ensemble is presently composed of eighteen 1 kg artefacts of various materials which have been chosen to minimize known or suspected sources of mass instability. A storage facility has been designed to minimize the rate of surface contamination of the artefacts. Technical details are provided in <<bipm-paper>>.
```

Comment:
I'm presuming this is not going to generate the text "[4.8]", and that that is acceptable. (It certainly is to me: what passes for citations in existing BIPM practice is unacceptably louche.)

Reply:
Actually, it is going to generate the text "[4.8]", because the references were manually renamed...

Reply:
Oh, right, I remember now. Cool.